File: 10871/helloworld.py
in_result = input("").split(" ")


#10 5
first_input = int(in_result[0])
second_input = int(in_result[1])
newArray = []

in_second_input = input("").split(" ")

for res in in_second_input:
    if int(res) < second_input:
        newArray.append(res)

print(" ".join(newArray))

File: 10871/helloworld.js
const fs = require('fs');
const input =  fs.readFileSync("/dev/stdin").toString().split(" ");
const secondLine =  fs.readFileSync("/dev/stdin").toString().split("\n")[1];

const firstInput = parseInt(input[0]);
const secondInput = parseInt(input[1]);
const array = [];

const loop = secondLine.split(" ");
if(loop.length === firstInput){
    for(let N in loop){
        if(loop[N] < secondInput){
            array.push(loop[N]);
        }
    }
}

console.log(array.join(" "));